src/test/java/com/bakdata/common_kafka_streams/integration/CleanUpTest.java

```diff
@@ -175,9 +178,7 @@ void shouldDeleteValueSchema(final SoftAssertions softly)
                 .build();
 
         this.kafkaCluster.send(sendRequest);
-        this.app.run();
-        Thread.sleep(TimeUnit.SECONDS.toMillis(TIMEOUT_SECONDS));
-        this.app.close();
+        this.runApp();
```

Comment:
why is close not needed anymore?

Reply:
It's still needed. I just extracted the three lines into a method.

Reply:
My bad